pkg/defaulting/externalcluster.go

```diff
@@ -28,6 +28,15 @@ import (
 // This function assumes that the KubermaticConfiguration has already been defaulted
 // (as the KubermaticConfigurationGetter does that automatically).
 func DefaultExternalClusterSpec(ctx context.Context, spec *kubermaticv1.ExternalClusterSpec) error {
+	// The import cluster feature doesn't expect any cloud spec as this imported cluster could be any type of
+	// Kubernetes cluster, thus there are no specs in regard of cloud nor infrastructure. Setting the cluster spec
```

Comment:
```suggestion
	// Kubernetes cluster, thus there are no specs in regard of cloud nor infrastructure. Setting the cloud spec
```

Reply:
I am setting the cluster spec here not only the cloud spec.